src/core/IODSExperimentalContextProvider.tsx

```diff
@@ -37,3 +37,19 @@ export const IODSExperimentalContextProvider = ({
     </IOExperimentalDesignContext.Provider>
   );
 };
+export const IODSExperimentalEnabledContextProvider = ({
+  children,
+  isExperimentaEnabled
+}: React.PropsWithChildren<IOExperimentalContextProviderProps>) => {
+  const [isExperimental, setExperimental] = React.useState(
+    isExperimentaEnabled ?? true
+  );
+
+  return (
+    <IOExperimentalDesignContext.Provider
+      value={{ isExperimental, setExperimental }}
+    >
+      {children}
+    </IOExperimentalDesignContext.Provider>
+  );
+};
```

Comment:
instead of creating a new instance of our context provider what about using the original one setting to `true` the `isExperimentaEnabled` prop ?

Reply:
```suggestion
```

Reply:
You are absolutely right! Thanks!